ui/app/AppLayouts/Chat/views/ActivityCenterGroupRequest.qml

```diff
@@ -40,7 +42,7 @@ Item {
                 return false
             }
 
-            return activityCenter.currentFilter === ActivityCenter.Filter.All
+            return acCurrentFilterAll;
```

Comment:
does an && to all apply here? if hideReadNotifications is false for example, it won't get into the if and it will just return acCurrentFilterAll, isn't it?